tests/benchmark/test_forecast_transferability_stress_matrix.py

```
@@ -0,0 +1,153 @@
+) -> None:
+    """CLI should write reviewable JSON and Markdown artifacts."""
+    metric_path = tmp_path / "metrics.json"
+    out_json = tmp_path / "transfer.json"
+    out_md = tmp_path / "transfer.md"
+    metric_path.write_text(
+        json.dumps(_metric_report(transfer_dimensions=_complete_transfer_dimensions())),
+        encoding="utf-8",
+    )
+
+    monkeypatch.chdir(Path(__file__).resolve().parents[2])
+    result = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)